doc/Advanced-configuration.md

```diff
@@ -247,7 +247,7 @@ See [Database backends configuration](./advanced-configuration/database-backends
 
 * **odbc_server** (local)
     * **Description:** SQL DB connection configuration. Currently supported DB types are `mysql` and `pgsql`.
-    * **Syntax:** `{odbc_server, {Type, Host, Port, DBName, Username, Password}}.`
+    * **Syntax:** `{odbc_server, {Type, Host, Port, DBName, Username, Password}}.` **or** ODBC connection string.
```

Comment:
This reads to me like `ODBC connection string` *replaces* the whole `{odbc_server, ...}` tuple